Makefile

```diff
@@ -246,7 +249,8 @@ test-e2e: test-e2e-wait-for-stable-state
 	-p 1 \
 	-tags e2e \
 	-run "$(TEST)" \
-	./test/e2e
+	./test/e2e \
+	-ginkgo.label-filter=$(E2E_GINKGO_LABEL_FILTER)
```

Comment:
Does addition of this skip the non-ginkgo tests present in `test/e2e/cert_manager_deployment_test.go`?
It seems fine to me if it does, because those tests are essentially repetition of what's also in the ginkgo suite so we can skip them during e2e runs moving forward (we have had this behaviour tested for long and it isn't a problem).

Reply:
This filter only applies to tests in the Ginkgo framework. The `test/e2e/cert_manager_deployment_test.go` go tests would not be affected.